hack/e2e/environment.sh

```diff
@@ -16,4 +30,16 @@ export IMAGE_CHECKSUM_TYPE="md5"
 # shellcheck disable=SC2016
 export CONTROL_PLANE_MACHINE_COUNT='${CONTROL_PLANE_MACHINE_COUNT}'
 # shellcheck disable=SC2016
-export WORKER_MACHINE_COUNT='${WORKER_MACHINE_COUNT}'
\ No newline at end of file
+export WORKER_MACHINE_COUNT='${WORKER_MACHINE_COUNT}'
+
+os_check
+
+if [[ "${OS}" == ubuntu ]]; then
+    export IMAGE_OS=${IMAGE_OS:-"Ubuntu"}
+    export CONTAINER_RUNTIME=${CONTAINER_RUNTIME:-"docker"}
+    export EPHEMERAL_CLUSTER=${EPHEMERAL_CLUSTER:-"kind"}
```

Comment:
We can drop these I believe, only `OS` provided should be enough to find out both [CONTAINER_RUNTIME](https://github.com/metal3-io/metal3-dev-env/blob/master/lib/common.sh#L58-L62) and [EPHEMERAL_CLUSTER](https://github.com/metal3-io/metal3-dev-env/blob/master/lib/common.sh#L201-L206) vars